test/screenshot/lib/image-cropper.js

```diff
@@ -94,21 +108,74 @@ class ImageCropper {
       if (!cols[x]) {
         cols[x] = [];
       }
-      const isMatch = jimpImage.getPixelColor(x, y) === this.trimColorInt_;
-      rows[y][x] = isMatch;
-      cols[x][y] = isMatch;
+      const pixelColorRGB = jimp.intToRGBA(jimpImage.getPixelColor(x, y));
+      const isTrimColor = this.isTrimColor_(pixelColorRGB);
+      rows[y][x] = isTrimColor;
+      cols[x][y] = isTrimColor;
     });
 
     return {rows, cols};
   }
 
   /**
+   * @param {!RGBA} pixelColorRGB
+   * @return {boolean}
+   * @private
+   */
+  isTrimColor_(pixelColorRGB) {
+    return (
+      Math.abs(pixelColorRGB.r - this.trimColorRGB_.r) < TRIM_COLOR_CHANNEL_DISTANCE &&
```

Comment:
If the only colors that change value across browser (based on your comment) are green and blue, do you need the red color distance check?

Reply:
I added it mainly out of paranoia - I don't trust the `red` color value to be the same on every device 😛